openscap/openscap.go

```diff
@@ -0,0 +1,105 @@
+package openscap
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+const (
+	CPE               = "oval:org.open-scap.cpe.rhel:def:"
+	MAX_DIST          = 7
+	MIN_DIST          = 5
+	CPE_DICT          = "/usr/share/openscap/cpe/openscap-cpe-oval.xml"
+	CVE_URL           = "https://www.redhat.com/security/data/metrics/ds/"
+	DIST_CVE_NAME_FMT = "com.redhat.rhsa-RHEL%d.ds.xml.bz2"
+	ARF_RESULT_FILE   = "results-arf.xml"
+)
+
+func getRHELDist(chrootPath string) (int, error) {
+	for dist := MIN_DIST; dist <= MAX_DIST; dist = dist + 1 {
+		output, err := oscapChroot(chrootPath, "oval", "eval", "--id",
+			fmt.Sprintf("%s%d", CPE, dist), CPE_DICT, "2>&1", ">", "/dev/null")
```

Comment:
@enoodle don't use all these bash redirections. After all you're getting both `output` and `err`.